core/src/main/scala/org/apache/spark/scheduler/TaskSet.scala

```diff
@@ -34,4 +37,12 @@ private[spark] class TaskSet(
   val id: String = s"$stageId.$stageAttemptId"
 
   override def toString: String = "TaskSet " + id
+
+  // Identifier used in the structured logging framework.
+  lazy val logId: MessageWithContext = {
```

Comment:
So we intentionally not treat it as a log parameter?

Reply:
what do you mean by "log parameter"?

Reply:
nvm, it's a `MessageWithContext`